docs/node-operators/archive-node.mdx

```diff
@@ -9,12 +9,17 @@ A new version of Mina Docs is coming soon! This page will be rewritten.
 
 :::
 
-
 # Archive Node
 
 Mina nodes are succinct by default, meaning they don't need to maintain historical information about the network, block, or transactions.
 For some usecases, it is useful to maintain this historical data, which you can do by running an **Archive Node.**
 
+:::tip
+
+A zkApp can retrieve events and actions from one or more Mina archive nodes. If your smart contract needs to fetch events and actions from an archive node, see [How to Fetch Events and Actions](../zkapps/advanced-snarkyjs/#fetch-events-and-actions).
```

Comment:
Just fyi. We can currently only fetch from one archive node, but we have plans to fetch from more than one simultaneously for redundancy, in the next sprint.

This is probably ok as is.